libopenage/engine.cpp

```diff
@@ -548,4 +542,12 @@ void Engine::move_phys_camera(float x, float y, float amount) {
 	this->engine_coord_data->camgame_phys += cam_delta.to_phys3();
 }
 
+void Engine::start_game(std::unique_ptr<GameMain> game) {
+	// TODO: maybe implement a proper 1-to-1 connection
+	assert(game);
```

Comment:
Hmm, I tend to disagree: Explicit is better than implicit, and the `MSG` macro does all the magic to track the originating line (btw what will happen if we have a generated cpp file via `moc`? is the line numbering converted as well?) and the exception is thrown through the native c++ mechanism. I hardly see a way of making it more minimal except having another magic macro as an alias.